reference/configuration/framework.rst

```diff
@@ -1094,7 +1094,66 @@ version_strategy
 **type**: ``string`` **default**: ``null``
 
 The service id of the :doc:`asset version strategy </frontend/custom_version_strategy>`
-applied to the assets.
+applied to the assets. This option can be set globally for all assets and
+individually for each asset package:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # app/config/config.yml
+       framework:
+            assets:
+                # this strategy is applied to every asset (including packages)
+                version_strategy: 'app.asset.my_versioning_strategy'
+                packages:
+                    foo_package:
+                        # this makes the assets of this package to not be versioned
+                        version: ~
+                    bar_package:
+                        # this package doesn't use the global versioning strategy
+                        version_strategy: 'app.asset.another_version_strategy'
```

Comment:
I would add a third part not redefining the version or strategy (but changing the base URL for instance), with a comment explaining that it inherits the default strategy, to make it easier to understand. What do you think about it ?